apps/s_cb.c

```diff
@@ -1074,7 +1074,7 @@ static void print_raw_cipherlist(SSL *s)
             BIO_puts(bio_err, ":");
         if (c)
             BIO_puts(bio_err, SSL_CIPHER_get_name(c));
-        else if (!memcmp(rlist, scsv_id - num + 3, num))
+        else if (num <= 3 && !memcmp(rlist, scsv_id - num + 3, num))
```

Comment:
You go it right:
- the test  'num <= 3' don't solve the issue.
- it's an operation order issue, not a real bug .